build.go

```diff
@@ -71,6 +74,22 @@ func Build(entryResolver EntryResolver,
 		if err != nil {
 			return packit.BuildResult{}, err
 		}
+		logger.Process("Checking custom npm version")
+		npmVersion, found := environment.Lookup("BP_NPM_VERSION")
+		if found {
+			args := []string{"install", "-g", fmt.Sprintf("npm@%s", npmVersion)}
```

Comment:
@c0d1ngm0nk3y do you know where all the globall installed npm version later used?  I know there are some cases where a global install has issues on linux install and I'm wondering if it can be installed non-global and then either direct paths or path search order can be used to get the installed non-global version?

Reply:
We changed it to a non global version